src/redux/configureStore.js

```diff
@@ -3,8 +3,8 @@ import booksReducer from './books/books';
 import categoriesReducer from './categories/categories';
 
 const rootReducer = combineReducers({
-  booksReducer,
-  categoriesReducer,
+  books: booksReducer,
+  categories: categoriesReducer,
 });
 
 const store = createStore(rootReducer);
```

Comment:
- [ ] <b>OPTIONAL:</b>  There are few thing you can do to improve your development experience with React/Redux.
   -  Install [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) and  [Redux DevTools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) from chrome extensions. and they both be accessed in inspect element panel of your page.
   -  You can also use `logger` to monitor your state in logs.
   - After that, install the following packages `npm i --save redux-logger redux-devtools-extension redux-thunk`
   - The change your code to reflect this
      ```js
        import thunk from 'redux-thunk';
        import { applyMiddleware, createStore } from 'redux';
        import { logger } from 'redux-logger';
        import { composeWithDevTools } from 'redux-devtools-extension';
        import reducer from './reducers';
  
       const { NODE_ENV } = process.env;

       const middleware = NODE_ENV === 'development'
         ? composeWithDevTools(applyMiddleware(thunk, logger))
         : applyMiddleware(thunk, logger);

       export default createStore(reducer, {}, middleware);

Reply:
@rukundoeric, thanks for the review and thanks for the tips 🙏🏽